README.md

```diff
@@ -65,7 +65,60 @@ Hence, we are trying to find and then follow good patterns in naming, code organ
 
 #### Releasing
 
-The standard release process is done manually.
+The standard release process is done manually. There are multiple
+repositories taking part in the process. The
+[renku](https://github.com/SwissDataScienceCenter/renku) project
+contains helm charts for deploying to kubernetes and the acceptance
+tests. The
+[terraform-renku](https://github.com/SwissDataScienceCenter/terraform-renku)
+project contains deployment descriptions for all environments.
+
+**renku-graph project:**
+- create a branch off the `origin/release` branch, name it
+  `prep-<version>`
+- either cherry-pick from development to here or do a `git rebase
+  origin/development`
+- make a PR for this branch to be merged into origin/release
+  ([example](https://github.com/SwissDataScienceCenter/renku-graph/pull/1095))
+- list some points in the description for what has been changed
+- merge the PR and create a release on github
+  - release description: list the points again but organized in
+    Features, Bugfixes etc
+    ([example](https://github.com/SwissDataScienceCenter/renku-graph/releases/tag/2.19.0))
+  - The release notes are public, so don't list refactorings.
```

Comment:
A super tiny thing. This only point starts with a capital _'T'_ so let's maybe align that with the rest.